src/CMakeLists.txt

```diff
@@ -192,22 +192,25 @@ add_executable(slade WIN32 MACOSX_BUNDLE
 	${SLADE_HEADERS}
 )
 
+
 target_link_libraries(slade
-	${ZLIB_LIBRARY}
-	${BZIP2_LIBRARIES}
-	${EXTERNAL_LIBRARIES}
-	${wxWidgets_LIBRARIES}
-	${FREEIMAGE_LIBRARIES}
-	${SFML_LIBRARY}
-	${FTGL_LIBRARIES}
-	${OPENGL_LIBRARIES}
-	${GLEW_LIBRARY}
-	${CURL_LIBRARIES}
-	${LUA_LIBRARIES}
-	${MPG123_LIBRARIES}
-	${fmt_LIBRARIES}
-	-lstdc++fs
+		${ZLIB_LIBRARY}
+		${BZIP2_LIBRARIES}
+		${EXTERNAL_LIBRARIES}
+		${wxWidgets_LIBRARIES}
+		${FREEIMAGE_LIBRARIES}
+		${SFML_LIBRARY}
+		${FTGL_LIBRARIES}
+		${OPENGL_LIBRARIES}
+		${GLEW_LIBRARY}
+		${CURL_LIBRARIES}
+		${LUA_LIBRARIES}
+		${MPG123_LIBRARIES}
+		${fmt_LIBRARIES}
```

Comment:
Please avoid useless changes in spacing like this altered indent and empty line. It’s much harder to review such changes. Git diff and blame features will produce unneeded noise.